src/emysql.erl

```diff
@@ -324,7 +322,7 @@ add_pool(PoolId, Size, User, Password, Host, Port, Database, Encoding) ->
       Database :: string(),
       Encoding :: utf8 | latin1 | {utf8, utf8_unicode_ci} | {utf8, utf8_general_ci},
       StartCmds :: list(binary()),
-      Result :: {reply, {error, pool_already_exists}, state()} | {reply, ok, state() }.
+      Result :: {error, pool_already_exists} | ok.
 add_pool(PoolId, Size, User, Password, Host, Port, Database, Encoding, StartCmds) ->
```

Comment:
Line 326 is too long: add_pool(PoolId, Size, User, Password, Host, Port, Database, Encoding, StartCmds) ->.